src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -96,6 +96,7 @@
 import org.lwjgl.opengl.GL13;
 
 import javax.annotation.Nonnull;
+
```

Comment:
No reason for this whitespace only change in this file.

Reply:
Agh, thanks for the catch.